README.md

```diff
@@ -29,3 +29,7 @@ Documentation viewers should be able to display all the relevant information abo
 ## Linting
 
 Linters should be able to produce warnings based on custom element defintions, such as warning if unknown elements are used in HTML templates.
+
+## Testing
+
+Developers should be able to automate tests to ensure the custom element API (attributes, properies, events) has not been changed.
```

Comment:
I'm not sure I follow what this means here. Can you elaborate?

Reply:
We can use the custom-element.json as a snapshot of the component api. If the test fail because of an attribute, property, event, etc... changed we can evaluate the impact and decide the version bump.

Reply:
This is actually a really nice usecase that I hadn't considered. iirc Elm has something similar where the publishing tool detects API changes, and prevents accidental breaking changes, I have always been super impressed by that kind of tooling.

Maybe we can change the wording here a bit to make things more clear:

```
Tooling would be able to detect whether or not the public API of a custom element has changed, based on a snapshot of the current `custom-elements.json` file to decide the impact of an update, and potentially prevent breaking API change in patch versions.
```

Reply:
@jorgecasar can you update this text here to be more detailed like what @thepassle wrote?

Reply:
Done!